### ISSUE TITLE
IronPython.StdLib is not loaded

### ISSUE THREAD

**Issue by Cyril-Pop** [DynamoDS/DSIronPython3#7]
Hi,

## Dynamo version

DynamoSanbox 3.1.0 

## IronPython version 

 3.4.1 

## DynamoIronPython3 Package version 

 1.2.0

## Operating system

Win 11

## What did you expect to see?

IronPython.StdLib is not loaded by default

![image](https://github.com/DynamoDS/DSIronPython3/assets/53660624/5d14180c-f490-4bed-ae41-67f11c75f171)

## code to reproduce 

```
import sys
import clr
try:
    import functools
except Exception as e:
    OUT = e, sys.version, sys.path
```

## workaround adding the correct **StdLib path** in `sys.path`

```
import sys
import clr
import System
clr.AddReference("System.Core")
clr.ImportExtensions(System.Linq)
#
ipy3_assembly = System.AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(lambda a : a.GetName().Name == 'IronPython.Modules')
assembly_ipy3_Directory = System.IO.Path.GetDirectoryName( ipy3_assembly.Location )
parent_ipy3_Directory = System.IO.Directory.GetParent(assembly_ipy3_Directory).FullName
#
sys.path.append(parent_ipy3_Directory + "\\lib")
#
try:
    import functools
    OUT = sys.version, functools, sys.path
except Exception as e:
    OUT = e, sys.version, sys.path
```
![image](https://github.com/DynamoDS/DSIronPython3/assets/53660624/d445b055-3045-48af-bcee-e412efa46653)

**Comment by mjkkirschner** [comment]
thanks for reporting @Cyril-Pop - the DynamoIronPython3 1.2 package structure is incorrect. I will try to resolve it and publish 1.3. Will close this issue when it's fixed.

**Comment by mjkkirschner** [comment]
@Cyril-Pop should be resolved in version 1.3.1 - please reopen if it's still an issue.
